packages/components/addon/components/hds/dropdown/index.js

```diff
@@ -35,6 +35,15 @@ export default class HdsDropdownIndexComponent extends Component {
     return listPosition;
   }
 
+  /**
+   * @param listAriaLabel
+   * @type {string}
+   * @description Sets a label for when  checkmark list items are used in the dropdown list
+   */
+  get listAriaLabel() {
+    return this.args.listAriaLabel || '';
```

Comment:
A comment and a consideration:
- should we have a meaningful default
- if there is a meaningful default we can use (eg. "List of options" or "A list containing a set of options") can we just use it directly without asking the consumer to provide an extra argument (they may not know, and we don't enforce it, so it's very likely it will be missed)

Reply:
@MelSumner when you have a moment to spare I would like to get your feedback on this change and potentially for a default label

Reply:
I would say to try to use the trigger button label. Give it an ID, and then put `aria-labelledby` on the `<ul>` with the `role="listbox"`, where the value of `aria-labelledby` is the ID of the `<button>` text used for the trigger. 

WDYT?

Reply:
That sounds appropriate – I'll work towards that

Reply:
@alex-ju and I played a little bit with the code and we found that `closest()` works (to be tested more, just to be sure) and so the solution proposed is OK

Reply:
I don't have any objection to this approach. 👍

Reply:
@didoo when time allows, I think this is good for another review